src/gemdat/volume.py

```diff
@@ -288,6 +288,26 @@ def to_structure(
 
         return structure
 
+    def get_free_energy(
+        self,
+        kBT,
+    ) -> np.ndarray:
+        """Estimate the free energy from volume.
+
+        Parameters
+        ----------
+        kBT : float
+            The temperature of the thermostat
+
+        Returns
+        -------
+        free_energy : ndarray
+            Free energy on the voxel grid
+        """
+        prob = self.data / self.data.sum()
+        free_energy = -kBT * np.log(prob)
+        return np.nan_to_num(free_energy)
```

Comment:
Maybe not for this PR, but for the long term we could consider that all methods like this return an instance of `Volume` with some flag that tells that it is a probability or free energy, and have some `.units` attribute to keep track of the units.